cmd/manager/main.go

```diff
@@ -4,14 +4,30 @@ import (
 	"fmt"
 	"os"
 
-	"github.com/mongodb/mongodb-kubernetes-operator/pkg/apis"
-	"github.com/mongodb/mongodb-kubernetes-operator/pkg/controller"
+	mdbv1 "github.com/mongodb/mongodb-kubernetes-operator/api/v1"
+	controllers "github.com/mongodb/mongodb-kubernetes-operator/controllers"
```

Comment:
I don't think there is a need to add `controllers` explicitly since it will be used from the import path